Document require.label config options (#830)
packit/packit.dev: docs/configuration/downstream/koji_build.md

   - name of a group - starting with `@`, e.g. `@my-sig`
   - `all_admins` alias - allowing all users with admin access to the dist-git repo
   - `all_committers` alias - allowing all users with commit access to the dist-git repo
* **require.label** - you can specify labels that have to be present/absent on a pull request

> so that merging of the pull request triggers the build. See configuration details [here](/docs/configuration#require).

Reviewer: ```suggestion
in order to trigger the build when it is merged. See configuration details [here](/docs/configuration#require).
```
